Packages/Sites/Sfi.Kateheo/Resources/Private/TypoScript/Root.ts2

```diff
@@ -29,8 +29,10 @@ prototype(Sfi.Kateheo:Page) < prototype(TYPO3.Neos:Page) {
 		}
 
 		@context.sectionMenu = Menu {
```

Comment:
It's better to put the whole preconfigured menu object into own TS object prototype, e.g.
```
prototype(Sfi.Kateheo:TwoLevelMenu) < prototype(TYPO3.Neos:Menu) {
...
}
```
And then use it as `@context.sectionMenu = Sfi.Kateheo:TwoLevelMenu {...`